src/components/Footer/Footer.js

```diff
@@ -175,12 +275,40 @@ const styles = StyleSheet.create({
     paddingRight: 13.5,
     paddingLeft: 18
   },
-  btnComment: {
+  btnDM: {
     height: '100%',
+    paddingLeft: dimen.normalizeDimen(20),
     justifyContent: 'center',
-    paddingLeft: 13.5,
-    paddingRight: 10
+    alignItems: 'center'
+  },
+  textDM: {
+    fontSize: normalizeFontSize(12),
+    fontWeight: '600',
+    color: colors.greySubtile1,
+    marginLeft: dimen.normalizeDimen(4),
+    textAlignVertical: 'center'
   },
+  card: {
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: colors.white,
+    width: dimen.normalizeDimen(50),
+    height: dimen.normalizeDimen(26),
+    elevation: dimen.normalizeDimen(2),
+    borderRadius: dimen.normalizeDimen(13),
+    padding: dimen.normalizeDimen(4),
+    shadowColor: 'rgba(0, 0, 0, 0.30)',
+    shadowOffset: {width: 0, height: 3},
+    shadowOpacity: dimen.normalizeDimen(0.7),
+    shadowRadius: dimen.normalizeDimen(2)
+  },
+  btnComment: (isShowDM, isSelf) => ({
+    height: '100%',
+    justifyContent: 'center',
+    paddingLeft: isShowDM && !isSelf ? 20 : 13.5,
+    paddingRight: isShowDM && !isSelf ? 6 : 10
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `btnComment` style has been updated to accommodate the new DM feature. Ensure that these changes do not negatively affect existing components that use this style.
```diff
- btnComment: {
-   height: '100%',
-   justifyContent: 'center',
-   paddingLeft: 13.5,
-   paddingRight: 10
- },
+ btnComment: (isShowDM, isSelf) => ({
+   height: '100%',
+   justifyContent: 'center',
+   paddingLeft: isShowDM && !isSelf ? 20 : 13.5,
+   paddingRight: isShowDM && !isSelf ? 6 : 10
+ }),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->